core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2717,73 +2603,48 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_NETWORK_NETTY_CHANNEL_POOL_SIZE_MAX =
-      new Builder(Name.USER_NETWORK_NETTY_CHANNEL_POOL_SIZE_MAX)
-          .setDefaultValue(1024)
-          .setDescription("The maximum number of netty channels cached in the netty channel "
-              + "pool.")
-          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
-          .setScope(Scope.CLIENT)
-          .build();
-  public static final PropertyKey USER_NETWORK_NETTY_CHANNEL_POOL_GC_THRESHOLD_MS =
-      new Builder(Name.USER_NETWORK_NETTY_CHANNEL_POOL_GC_THRESHOLD_MS)
-          .setAlias(new String[]{"alluxio.user.network.netty.channel.pool.gc.threshold.ms"})
-          .setDefaultValue("300sec")
-          .setDescription("A netty channel is closed if it has been idle for more than this "
-              + "threshold.")
+  public static final PropertyKey USER_NETWORK_READER_BUFFER_SIZE_MESSAGES =
+      new Builder(Name.USER_NETWORK_READER_BUFFER_SIZE_MESSAGES)
+          .setDefaultValue(16)
+          .setDescription("When a client reads from a remote worker, the maximum number of "
+              + "messages to buffer by the client.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_NETWORK_NETTY_CHANNEL_POOL_DISABLED =
-      new Builder(Name.USER_NETWORK_NETTY_CHANNEL_POOL_DISABLED)
-          .setDefaultValue(false)
-          .setDescription("Disable netty channel pool. This should be turned on if the client "
-              + "version is >= 1.3.0 but server version is <= 1.2.x.")
+  public static final PropertyKey USER_NETWORK_READER_CHUNK_SIZE_BYTES =
+      new Builder(Name.USER_NETWORK_READER_CHUNK_SIZE_BYTES)
+          .setDefaultValue("64KB")
+          .setDescription("When a client reads from a remote worker, the maximum chunk size.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
-  public static final PropertyKey USER_NETWORK_NETTY_WRITER_BUFFER_SIZE_PACKETS =
-      new Builder(Name.USER_NETWORK_NETTY_WRITER_BUFFER_SIZE_PACKETS)
+  public static final PropertyKey USER_NETWORK_WRITER_BUFFER_SIZE_MESSAGES =
+      new Builder(Name.USER_NETWORK_WRITER_BUFFER_SIZE_MESSAGES)
           .setDefaultValue(16)
-          .setDescription("When a client writes to a remote worker, the maximum number of packets "
+          .setDescription("When a client writes to a remote worker, the maximum number of messages "
```

Comment:
what is a message, what is a chunk?

Reply:
message can be either a request command, a data chunk, or a gRPC stream event that is pushed to the receiving end.